src/Common/src/Internal/Runtime/ModuleHeaders.h

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I agree - we should follow the same convention as what e use for other .h/.cs files.